bentoml/yatai/client/bento_repository_api.py

```diff
@@ -200,21 +267,38 @@ def upload_from_dir(self, saved_bento_path, labels=None):
             )
 
         if response.uri.type == BentoUri.LOCAL:
-            if os.path.exists(response.uri.uri):
-                # due to copytree dst must not already exist
-                shutil.rmtree(response.uri.uri)
-            shutil.copytree(saved_bento_path, response.uri.uri)
-
-            self._update_bento_upload_progress(bento_service_metadata)
+            # For remote yatai service, call upload method.
+            if isinstance(self.yatai_service, YataiStub):
+                try:
```

Comment:
Let's just have `self.upload_bento` return the UploadStatus here?  It seems weird to me to catch a generic Exception here for this